doc/source/contribution/documentation/introduction.rst

```diff
@@ -6,6 +6,19 @@ Introduction
 This documentation describes about how to write, generate and manage
 Groonga documentation.
 
+How to clone Groonga repository
+-------------------------------
+
+First, please fork Groonga repository on GitHub because
+you need to build Groonga documentation from the latest source code.
+You just access https://github.com/groonga/groonga and press
+`Fork` button. Now you can clone your Groonga repository.
+
+.. code-block:: console
+
+  % git clone https://github.com/${YOUR_GITHUB_ACCOUNT}/groonga.git
```

Comment:
```suggestion
  % git clone --recursive https://github.com/${YOUR_GITHUB_ACCOUNT}/groonga.git
```

Reply:
Thanks mm, I forgot the most important option here.

Reply:
fix: 3592e94 I added --recursive option because we need submodules for building Groonga.